### PR TITLE
transfer f0 to semitone

### PR DESCRIPTION
**Pull request by PeterGuoRuc**

### CODE REVIEW

**Comment by ftshijt** [comment]
you can remove wip after your tests, then I will go through the code

**Comment by PeterGuoRuc** [comment]
> you can remove wip after your tests, then I will go through the code
I have checked the code & I will remove it.

File: SVS/bin/infer.py [hunk 1]
```diff
@@ -48,16 +51,28 @@
         help="Combine multiple datasets & add singer embedding",
     )
     parser.add_argument(
-        "--random_crop",
```
Comment (ftshijt) [inline SVS/bin/infer.py:51]: why remove random crop here

Comment (PeterGuoRuc) [reply SVS/bin/infer.py:51]: There is no need for random_crop during the test period. (No augmentation for test set !

**Pull request merged**